### PR TITLE
Add tables for ecosystem support and workarounds

### PR DESCRIPTION
**Pull request by scottmalkie**
*Issue #, if available:* N/A

*Description of changes:*
Add table in containers.md for explicit ecosystem support
Add containers-workarounds.md with details on workarounds

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: containers-workarounds.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+| Name                      | URL / Github issue            | Workaround             | Existing image? |
+| :-----                    |:-----                         | :-----                 | :-----          |
+| ALB Ingress Controller | https://github.com/kubernetes-sigs/aws-alb-ingress-controller/issues/1329	| compile from source	| alittlec/aws-alb-ingress-controller:v1.1.8 |
+| FireLens for Amazon ECS | https://github.com/aws/aws-for-fluent-bit/issues/44 | compile from source | |
+| ECS CLI | https://github.com/aws/amazon-ecs-cli/pull/1066 | use PR branch | |
+| cluster-autoscaler | https://github.com/kubernetes/autoscaler/issues/3419 | compile from source or use PR branch | raspbernetes/cluster-autoscaler |
+| external-dns | https://github.com/kubernetes-sigs/external-dns/issues/1721 |	compile from source | raspbernetes/external-dns	|
+|metrics-server	| https://github.com/kubernetes-sigs/metrics-server	| build via helm chart |k8s.gcr.io/metrics-server-arm64	|
+| kube-state-metrics | https://github.com/kubernetes/kube-state-metrics/pull/1190 |compile from source | alittlec/kube-state-metrics-arm64 |
```
Comment (AWSNB) [inline containers-workarounds.md:18]: i believe this one is merged already

File: containers-workarounds.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+| kube-bench | | compile from source | raspbernetes/kube-bench |
+| AWS EBS CSI driver | https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/521 | compile from source or use PR branch	|chengpan/aws-ebs-csi-driver	|
+| AWS EFS CSI driver | https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/197| use PR branch | |
```
Comment (AWSNB) [inline containers-workarounds.md:20]: this seems a simple one for fix,  why couldn't we push it through

File: containers-workarounds.md (hunk 1 above)
Comment (scottmalkie) [reply containers-workarounds.md:18]: Thanks, correct issue is https://github.com/kubernetes-sigs/external-dns/issues/1443, will update.

File: containers-workarounds.md (hunk 2 above)
Comment (scottmalkie) [reply containers-workarounds.md]: We are working with the team to get them merged/

**Pull request merged**